source/chapters/configuration.rst

```diff
@@ -183,3 +183,92 @@ already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
 .. warning:: On GNU/Linux do *not* use the ``pulse`` device with the ALSA Audio
              API. This is an emulation layer for ALSA provided by PulseAudio and
              results in very poor performance. 
+             
+Equalizer Preferences
+=====================
+
+.. figure:: ../_static/Mixxx-111-Preferences-Equalizer(TODO).png
+   :align: center
+   :width: 80%
+   :figwidth: 100%
+   :alt: Equalizer Preferences
+   :figclass: pretty-figures
+
+   Equalizer Preferences
+   
+:menuselection:`Preferences --> Equalizer` allows you to setup the equalizers.
+
+* **Equalizer Rack**: The Equalizer Rack is a special Effect Rack, that is
+  connected to the deck's equalizer and filter controls. 
+
+  In this section you can select the equalizers and quick effects that used 
+  with the decks.    
+
+* **Equalizer Plugin**: Here you can select the effect that is uses as mixing 
+  EQ in each deck. By default only build in equalizers are displayed, but you 
+  are allowed to select any effect after you have unchecked "Only allow EQ 
+  knobs to control EQ specific effects"      
+
+  If you need different settings for each deck, you may uncheck "Use the same 
+  EQ filter for all decks"
+
+* **Quick Effect**: Here you can select the effect that is controlled by the 
+  filter knob in each deck. By default only build in filter effects are 
+  displayed in a common box for all decks, but it can be changed like above. 
+
+* **High Shelf EQ**: This slider sets the turnover frequency of the high shelf 
+  EQ and the upper corner frequency of the mid EQ. Depending on the selected 
+  EQ type, changing this slider during playback may result in a crackling sound. 
+
+* **Low Shelf EQ**: This slider sets the turnover frequency of the low shelf EQ
+  and the lower corner frequency of the mid EQ. Depending on the selected EQ 
+  type, changing this slider during playback may result in a crackling sound. 
+
+* **Master EQ**: This section allows you to setup an EQ that effects only the 
+  master output. This is useful to Equalize the sound of you sound system for 
+  the room, you play it.  
+
+
+Mixing Equalizers 
+-----------------
+
+Mixxx offers three types of build in mixing equalizers, also called isolators 
+or cross over since they are all full kill types. They have all their individual 
+sound, so try them out. 
```

Comment:
Are the equalizers called **isolators** or **cross over** anywhere else in the manual.  Is that sentence really important. For me (naive person without much knowledge in equalizers) the following 2 sentences would be enough.

`Mixxx offers three build in mixing equalizers. They all have their individual sound, to try them out`